### PR TITLE
feat: Cron job to gather Github info and digest them to a SQLite database

### PR DESCRIPTION
**Pull request by ZibanPirate**
- A cron gather Github information about repos listed in dzcode, and digest it into database.
- we then use this database instead of always querying Github, which is both slow and fragile.

#### Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (changes that require a fresh clone of the repo)
- [ ] I'm not sure

### CODE REVIEW

File: api/src/project/repository.ts [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+import { Service } from "typedi";
+
+import { ProjectRow, projectsTable } from "./table";
+
+@Service()
+export class ProjectRepository {
+  constructor(private readonly sqliteService: SQLiteService) {}
+
+  public async findForList() {
+    // @TODO-ZM: reverse hierarchy instead here
```
Comment (ZibanPirate) [inline api/src/project/repository.ts:15]: sort by contributions

File: packages/models/src/contributor/index.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+export interface ContributorEntity {
+  // @TODO-ZM: move this to BaseEntity
```
Comment (ZibanPirate) [inline packages/models/src/contributor/index.ts:2]: move it to BaseEntity

File: packages/models/src/language/index.ts [hunk 3]
```diff
@@ -8,14 +8,14 @@ export const allLanguages = [
 
 export class LanguageEntity extends BaseEntity {
```
Comment (ZibanPirate) [inline packages/models/src/language/index.ts:9]: convert to interface

File: packages/models/src/project-reference/index.spec.ts
Comment (ZibanPirate) [inline packages/models/src/project-reference/index.spec.ts:1]: remove project and repo reference models

File: web/src/pages/contribute/index.tsx [hunk 4]
```diff
@@ -66,17 +70,14 @@ export default function Page(): JSX.Element {
                           {language}
                         </span>
                       ))}
-                    </div>
+                    </div> */}
```
Comment (ZibanPirate) [inline web/src/pages/contribute/index.tsx:73]: remove langs and labels for now

File: api/package.json
Comment (ZibanPirate) [inline api/package.json:1]: remove dead models

**Review by souhaib-benbouzid — APPROVED** [review]

**Pull request merged**